packages/scanner/shared/lib/src/scanner.dart

```diff
@@ -43,9 +43,7 @@ abstract class CameraScanner with CameraScannerLogMixin {
   Future<void> onResume() async {}
 
   @mustCallSuper
-  Future<void> onDispose() async {
-    disposeLogs();
-  }
+  Future<void> onDispose() async => disposeLogs();
```

Comment:
That's problematic to override a `@mustCallSuper` method without calling its `super` method.

Reply:
The `@mustCallSuper` is for this override, not its parent

Reply:
Fair enough, my bad.